test/src/test/testtool/ResultSet2FileSender/01/xqs-in2.xml

```diff
@@ -1,6 +1,35 @@
 <root>
 	<select>
 		<tableName>IBISTEMP</tableName>
+		<columns>
+			<column>
+				<name>TKEY</name>
+			</column>
+			<column>
+				<name>TCHAR</name>
+			</column>
+			<column>
+				<name>TVARCHAR</name>
+			</column>
+			<column>
+				<name>TNUMBER</name>
+			</column>
+			<column>
+				<name>TTIMESTAMP</name>
+			</column>
+			<column>
+				<name>TDATE</name>
+			</column>
+			<column>
+				<name>TBLOB1</name>
+			</column>
+			<column>
+				<name>TCLOB</name>
+			</column>
+			<column>
+				<name>TBLOB2</name>
+			</column>
+		</columns>
```

Comment:
Wouldn't it be possible to to modify some outcomes slightly, and keep these queries simple?

Reply:
It is possible. However, any other change just like this one in the ibistemp table will result in updating the ouput files again. Selecting only the columns that are required for a scenario will not require an update in the output file.